.github/workflows/tests.yml

```diff
@@ -48,7 +46,7 @@ jobs:
           CXX: "sccache clang++"
 
   forest-sync-check:
-    if: startsWith(github.ref, 'refs/tags/v')
+    if: ${{ contains(github.event.pull_request.title, 'Release') }}
```

Comment:
I'm not 100% sure this is the right way. Perhaps using labels and labeling the release PRs would be more robust. No strong opinion, though.

In any case, we should create a list of steps needed to make a release in this repository, similar to the one that is there in Forest.

Reply:
switched to using `Release` label, and also added a publish and release workflow to publish all crates when a tag v* is pushed and create a pre-release with default content for that v*